docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -203,19 +144,70 @@ Reduce your attack surface by exposing only what you must.
             - { path: ^/search, roles: ROLE_USER}
     ```
 
-## Underlying stack
+## Symfony
 
-Once you have properly configured secure user roles and permissions, to avoid exposing your application to any DDOS vulnerabilities or other yet unknown security threats, make sure that you do the following:
+### `APP_SECRET`
 
-- Avoid exposing servers on the open internet when not strictly required.
-- Ensure any servers, services, ports and virtual hosts that were opened for testing purposes are shut down before going live.
-- Ensure file system permissions are set up in such a way that the web server or PHP user can't access files they shouldn't be able to read.
-- Secure the database with a good password, keys, firewall.
-Optionally, ensure that the database user used by the web app only has permissions to do the operations needed by [[= product_name =]].
-The Data Definition Language (DDL) commands (create, alter, drop, truncate, comment) are only needed for installing and upgrading [[= product_name =]], and not for running it.
-Not granting these rights to web app users reduces the damage that can result from a security breach.
+`APP_SECRET` needs to be a strong, random, securely stored value.
 
-Those steps aren't needed when using [[= product_name_cloud =]], where the provider handles them.
+- Don't use a default value like `ff6dc61a329dc96652bb092ec58981f7` or `ThisTokenIsNotSoSecretChangeIt`.
+- The secret must be secured against unwanted access. Don't commit the value to a version control system.
+- The secret must be long enough. 32 characters is minimum, longer is better.
+
+!!! tip
+
+    The following command generates a 64-character-long secure random value:
+
+    `php -r "print bin2hex(random_bytes(32));"`
+
+!!! note
+
+    On [[= product_name_cloud =]], if `APP_SECRET` isn't set, the system sets it to [`PLATFORM_PROJECT_ENTROPY`](https://docs.platform.sh/guides/symfony/environment-variables.html#symfony-environment-variables)
+
+### Symfony production mode
+
+Only expose Symfony production mode openly on the internet.
+Don't expose the dev mode on the internet, otherwise you may disclose things like `phpinfo` and environment variables.
+Exposing the dev mode exposes things like `phpinfo`, environment variables, and more.
```

Comment:
```suggestion
Don't expose the dev mode on the internet, otherwise you may disclose things like `phpinfo`, environment variables, and more.
```

Reply:
@dabrt This was resolved earlier in a different way, let me know if you still want it changed.